js/custom.js

```diff
@@ -7,59 +7,68 @@
   });
 })();
 
+var minutes = 0;
+setInterval(function () {
+  var timePlaceholderNode = document.getElementById('time_on_site');
+  if (timePlaceholderNode) {
+    minutes++;
+    var textContent = minutes + ' ' + (minutes > 1 ? 'minutes': 'minute') + ' on site';
+    timePlaceholderNode.textContent = textContent;
+  }
+}, 1000 * 60);
+
 var ARROW_UP_KEY = 38;
 var ARROW_DOWN_KEY = 40;
 var ARROW_RIGHT_KEY = 39;
 var ARROW_LEFT_KEY = 37;
 
-document.querySelectorAll("#nav li").forEach(function (navEl, _, parent) {
-  navEl.onclick = function () {
-    toggleTab(this.id, this.dataset.target);
-  };
-  navEl.onkeydown = function (event) {
-    var id = parseInt(this.id);
-    var target;
-    var resultId;
+var tabMenu = document.getElementById("nav");
+var liElements = document.querySelectorAll("#nav li");
+tabMenu.onclick = function (event) {
+  var listElement = event.target.closest("[data-target]");
+  if (listElement) {
+    toggleTab(listElement.id, listElement.dataset.target);
+  }
+}
+
+tabMenu.onkeydown = function (event) {
+  var listElement = event.target.closest("[data-target]");
+  if (listElement) {
+    var id = parseInt(listElement.id);
+    var liElementsLength = liElements.length;
     switch (event.keyCode) {
```

Comment:
keyCode is deprecated. Please use https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code instead.